### PR TITLE
fix tooltip description issue

### PR DESCRIPTION
**Pull request by cbadr-ext**
## Description

It seems that there is a mistake in GUI, on the description of the option “Service Check Timeout”, from page (Configuration  >  Pollers  >  Engine configuration), tab “Log options”.

The description should refer to UNKNOWN status instead of CRITICAL : 

image

**Fixes** # MON-15303

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

change tooltip description 

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by dmyios — DISMISSED** [review]

File: centreon/www/include/configuration/configNagios/help.php [hunk 1]
```diff
@@ -424,7 +424,7 @@
 $help["service_check_timeout"] = dgettext(
     "help",
     "This is the maximum number of seconds that Monitoring Engine will allow service checks to run. " .
-    "If checks exceed this limit, they are killed and a CRITICAL state is returned. A timeout error will " .
+    "If checks exceed this limit, they are killed and a UNKNOWN state is returned. A timeout error will " .
```
Comment (bdauria) [inline centreon/www/include/configuration/configNagios/help.php:427]:
```suggestion
    "If checks exceed this limit, they are killed and an UNKNOWN state is returned. A timeout error will " .
```

File: centreon/www/include/configuration/configNagios/help.php [hunk 2]
```diff
@@ -424,7 +424,7 @@
 $help["service_check_timeout"] = dgettext(
     "help",
     "This is the maximum number of seconds that Monitoring Engine will allow service checks to run. " .
-    "If checks exceed this limit, they are killed and a CRITICAL state is returned. A timeout error will " .
+    "If checks exceed this limit, they are killed and an UNKNOWN state is returned. A timeout error will " .
```
Comment (kduret) [inline centreon/www/include/configuration/configNagios/help.php:427]: translation must be updated in `lang/<locale>/LC_MESSAGES/help.po`

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/help.po [hunk 3]
```diff
@@ -1369,8 +1369,8 @@ msgstr ""
 #: centreon/www/include/configuration/configNagios/help.php:500
 msgid ""
 "This is the maximum number of seconds that Monitoring Engine will allow "
```
Comment (cg-tw) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/help.po:1371]:
```suggestion
"This is the maximum number of seconds that the monitoring engine will allow "
```

Comment (cg-tw) [reply centreon/lang/de_DE.UTF-8/LC_MESSAGES/help.po:1371]: the end of the toolitip must be corrected too. (If a service check runs longer than this limit, the monitoring engine will kill it off, thinking it is a runaway process."

**Review by cg-tw — DISMISSED** [review]

**Review by lpinsivy — APPROVED** [review]

**Review by lpinsivy — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Review by emabassi-ext — APPROVED** [review]

**Review by adr-mo — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**